bool loaded 
float64 position
int32 pathid
int32 cartid
